sdk/textanalytics/azure-ai-textanalytics/README.md

```diff
@@ -262,6 +264,22 @@ textAnalyticsClient.recognizeEntities(document).forEach(entity ->
 For samples on using the production recommended option `RecognizeEntitiesBatch` see [here][recognize_entities_sample].
 Please refer to the service documentation for a conceptual discussion of [named entity recognition][named_entity_recognition].
 
+### Recognize Personally Identifiable Information entities
+Run a predictive model to identify a collection of Personally Identifiable Information(PII) entities in the passed-in 
+document. It recognizes and categorizes PII entities in its input text, such as
+Social Security Numbers, bank account information, credit card numbers, and more. This endpoint is only available for 
+v3.1-preview.1 and up.
+
+<!-- embedme ./src/samples/java/com/azure/ai/textanalytics/ReadmeSamples.java#L158-L161 -->
+```java
+String document = "My SSN is 555-55-5555";
+textAnalyticsClient.recognizePiiEntities(document).forEach(entity -> System.out.printf(
+    "Recognized Personally Identifiable Information entity: %s, entity category: %s, entity subcategory: %s, offset: %s, length: %s, confidence score: %f.%n",
```

Comment:
Didn't check what we are following now for the readme samples. But based on our last discussions these examples should be concise and hence can remove info about length, offset, confidence score.
What do you think?

Reply:
Agreed. will update offset, length and confidence score in codesnippets

Reply:
+1 on always having confidence scores

Reply:
Offset and length still present?

Reply:
looking at the older commit.